.cargo/config.toml

```diff
@@ -1,2 +1,2 @@
 [env]
-CLASSPATH = { value = "java", relative = true, force = true }
+CLASSPATH = { value = "test-crates/duchess-java-tests/target/java", relative = true, force = true }
```

Comment:
I tried playing with this. Giving a value to `CLASSPATH` is apparently required, but the actual value is irrelevant. For example, `value = "inexistent"` would also work.

What about we remove this file, adapting the `javap` call to not pass `-cp` when `CLASSPATH` is unset?
1. In [`reflect.rs`](https://github.com/duchess-rs/duchess/blob/43c6bc5afe77f3ea4007a7ff8e86679d02b31055/macro/src/reflect.rs#L141-L151), instead of panicking we should still execute the command, but without passing `.arg("-cp").arg(classpath)`.
2. After that change, removing this `.cargo/config.toml` file should no longer break `cargo build`.